dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskExecutionContext.java

```diff
@@ -81,6 +81,11 @@ public class TaskExecutionContext implements Serializable {
      */
     private String logPath;
 
+    /**
+     * applicationId path
+     */
+    private String appInfoPath;
```

Comment:
It's no need to add this field if the appInfoPath is default `execPath/appInfo.log`.

Reply:
@caishunfeng 
You're right, no need, but it just seems more clear. Personally, I prefer to keep it, WDYT?